MahApps.Metro/Controls/Dialogs/ProgressDialogController.cs

```diff
@@ -12,23 +12,36 @@ public class ProgressDialogController
         private ProgressDialog WrappedDialog { get; set; }
         private Func<Task> CloseCallback { get; set; }
 
-        /// <summary>
-        /// Gets if the wrapped ProgressDialog is open.
-        /// </summary>
-        [Obsolete("Use the Closed event instead")]
-        public bool IsOpen { get; private set; }
+       
 
         /// <summary>
-        /// This event is raised when the associated <see cref="ProgressDialog"/> was closed.
+        /// This event is raised when the associated <see cref="ProgressDialog"/> was closed programmatically.
         /// </summary>
         public event EventHandler Closed;
+   
+        /// <summary>
+        /// This event is raised when the associated <see cref="ProgressDialog"/> was cancelled by the user.
+        /// </summary>
+        public event EventHandler Cancelled;
+
+        /// <summary>
+        /// Gets if the Cancel button has been pressed.
+        /// </summary>        
+        public bool IsCancelled {get{ return m_iscanceled;}}
```

Comment:
this can be simplyfied to `public bool IsCancelled { get; private set; }`
and i would prefere `IsCanceled`